test/parallel/test-fs-open-flags.js

```diff
@@ -1,84 +0,0 @@
-// Copyright Joyent, Inc. and other Node contributors.
```

Comment:
Would you mind not renaming this file. It makes the diff a lot nosier than it needs to be.